dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/InMemoryDocumentStore.kt

```diff
@@ -0,0 +1,37 @@
+package org.dataland.documentmanager.services
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+/**
+ * A service that can be used to store binary blobs (documents) in memory
+ */
+@Component
+class InMemoryDocumentStore {
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val dataInMemoryStorage = mutableMapOf<String, ByteArray>()
+
+    /**
+     * Stores the provided dataset in memory identified by its sha256 hash.
+     */
+    fun storeDataInMemory(hash: String, data: ByteArray): String {
+        dataInMemoryStorage[hash] = data
+        logger.debug("Stored blob with hash $hash to the in-memory store")
+        return hash
```

Comment:
Das verstehe ich nicht. Warum wird der hash, der ein Input-Parameter ist, wieder returned?

Reply:
🤷 removed that